operations/README.md

```diff
@@ -82,6 +82,12 @@ This section describes all operations supported by the API, organised here by th
 | [Add alternative payment](payments.md#add-alternative-payment) | Adds a new alternative payment to a specified customer |
 | [Add outlet bills](outletbills.md#add-outlet-bills) | Adds new outlet bills with their items |
 
+## Loyalty Programs
```

Comment:
Again as per previous comment, I think it should be "Loyalty programs" not "Loyalty Programs", like for example "Device integrations"